PocketKit/Sources/SaveToPocketKit/Services.swift

```diff
@@ -10,11 +10,24 @@ struct Services {
     let saveService: PocketSaveService
 
     private let persistentContainer: PersistentContainer
+    
+    private var subscriptions: Set<AnyCancellable> = []
 
     private init() {
+        Crashlogger.start(dsn: Keys.shared.sentryDSN)
         persistentContainer = .init(storage: .shared)
 
         appSession = AppSession()
+        
+        appSession.$currentSession.sink { session in
+            if let session = session {
+                Crashlogger.setUserID(session.userIdentifier)
+            } else {
+                Crashlogger.clearUser()
+            }
+        }.store(in: &subscriptions)
+        
+        
```

Comment:
The purpose of this `Services` object is to setup and keep ahold of various service objects that need to be used/injected throughout the rest of the app. I think this type of observation code would fit better in one of the view models, probably the `MainViewModel`.